Server/serverclient.py

```diff
@@ -0,0 +1,44 @@
+import socket
+import time
+
+time_string = ("%H:%M:%S")
+
+
+def time_now():
+    return time.strftime(time_string, time.localtime())
+
+
+class Server:
+    def __init__(self):
+        self.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        self.server.bind(('', 9009))
+        print(f'[SERVER STARTED {time_now()}]!')
```

Comment:
get timenow directly in method, don't use global variables

Reply:
fixed